Pipfile

```diff
@@ -75,7 +74,8 @@ types-six="*"
 types-PyMySQL="*"
 types-protobuf="*"
 typing-extensions="*"
-
+klara = "==0.6.3"
```

Comment:
why do we use a locked version here?

Reply:
this pr was just for testing, I think you should add the comment to eyal's PR